### DISCUSSION TITLE
AITHINKER CAM - tasmota32-webcam.bin MQTT Topic

### DISCUSSION THREAD

**Discussion by joednemesis** [arendst/Tasmota#14697] [Q&A] [answered]
I wrote the tasmota32-webcam.bin version 2022.01.3 on an AITHINKER CAM. Everything actually works, but the MQTT full topic is not formed correctly.

It says on the info page
![grafik](https://user-images.githubusercontent.com/27433206/152055998-03e67a11-b0f1-4cd3-9029-41cc9840830b.png)

I expected
![grafik](https://user-images.githubusercontent.com/27433206/152056051-6a561c07-a84e-4a38-9d52-b7325c317fa1.png)

and also in the console I see %topic% instead of the configured MQTT topic "esp32-01"

**Comment by barbudor** [comment]
Thanks for reporting
I had a similar issue 1 hour ago and then it disappeared
Have you tried to restart to see if it comes back to normal ?

**Reply by barbudor** [reply]
~~Note that your full topic is wrong
It must include `%prefix%` such as `%prefix%/%topic%` which is the default~~ 

Sorry this is actually what is on the Info page, not the settings

**Reply by joednemesis** [reply]
Restart didn't help. But what I also noticed: The web server generally reacts terribly sluggishly.

Downloading the stable version fixed the MQTT problem. 
But the web server is also very sluggish with the stable

**Comment by barbudor** [comment]
Did you built the firmware yourself or did you downloaded the one on http://ota.tasmota.com/tasmota32 ?

**Reply by barbudor** [reply]
We are investigating

**Reply by joednemesis** [reply]
Download

**Reply by barbudor** [reply]
You can get a binary from there : https://github.com/tasmota/install/tree/c6a76781ab35548937c868f286b9bd17e47d452e/firmware/development

These are the archive of development build from yesterday evening wiht Espressif framework 2.0.2.1

Since upgrade to Espressif framework 2.0.2.2, all binaries build on Feb 1st seems to have the problem

**Comment by Jason2866** [comment, marked as answer]
Issue fixed. https://github.com/tasmota/arduino-esp32/pull/63
All actual developmemt builds are updated and include the fix
